cmake/BundledCivetWeb.cmake

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+function(use_bundled_civetweb SOURCE_DIR BINARY_DIR)
+    message("Using bundled civetweb")
+
+    # Define patch step
+    set(PC "${Patch_EXECUTABLE}" -p1 -i "${SOURCE_DIR}/thirdparty/civetweb/civetweb.patch")
+
+    # Define byproducts
+    if (WIN32)
+        set(SUFFIX "lib")
+    else()
+		set(PREFIX "lib")
+        set(SUFFIX "a")
+    endif()
+
+    set(BYPRODUCTS
+            "lib/${PREFIX}civetweb.${SUFFIX}"
```

Comment:
Does prefix evaluates to an empty string in case of Win32?

Reply:
Yep, a non-existing variable converts to an empty string.